dcos/http.py

```diff
@@ -83,7 +83,8 @@ def request(method,
             request.headers)
 
         with requests.Session() as session:
-            response = session.send(request.prepare(), timeout=timeout)
+            response = session.send(request.prepare(), timeout=timeout,
+                                    proxies=util.get_proxy_dict_from_env())
```

Comment:
I don't think this is ncessary.  `requests` should understand the ENV vars w/o explicitly setting `proxies`